Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -259,5 +260,50 @@ protected void CheckMixedRealityInputActions()
                 }
             }
         }
+
+        /// <summary>
+        /// Applies the most relevant settings of DataProviderConfigurations
+        /// </summary>
+        /// <param name="typeName">name of the Type of the Data Provider</param>
+        /// <param name="providerAttribute"> the corresponding attribute</param>
+        /// <param name="providerName"> the target provider Name</param>
+        /// <param name="configurationProfile">the target configuration profile</param>
+        /// <param name="runtimePlatform">the target runtime plaform</param>
+        protected void ApplyDataProviderConfiguration(
```

Comment:
I might be missing something, but is this called anywhere?

Reply:
Looks like in the original PR, it was refactored out of MixedRealityInputSystemProfileInspector.cs, but that script isn't here

Reply:
per @Troy-Ferrell, this method is unused and has been removed.